src/news/views.py

```diff
@@ -84,6 +84,7 @@ def __mapItemToResult(item):
         'title': item.title,
         'tags': [item.name for item in item.newscategory.all()],
         'date': str(item.pub_date),
+        'display_date': str(item.pub_date.isoformat()),
```

Comment:
Could you please add a comment that explains that this field is going to be deprecated sometime next year (and open up a PR for it, but of course don't merge it until then)

Reply:
Why do we plan to deprecate `display_date`? 🤔

Reply:
Ops, wrong line! Of course the `date` field. I'm sorry about that! :/